pandas/tests/groupby/test_groupby.py

```diff
@@ -2116,6 +2116,38 @@ def interweave(list_obj):
             exp = DataFrame({'key': keys, 'val': _exp_vals})
             assert_frame_equal(result, exp)
 
+    @pytest.mark.parametrize("test_series", [True, False])
```

Comment:
move these to test_transform.py (didn't pay attention in previous fill PR of test locations, but should be the same)